.gitignore

```diff
@@ -31,3 +31,4 @@ modules/*
 .bundle/
 Gemfile.lock
 config/foreman.migrations/.applied
+.vendor/
```

Comment:
Where does this folder come from? And should it also be in the `AllCops/Exclude` list if that's where `bundle install` installs stuff.

Reply:
Rubocop seems to ignore hidden directories by default.